app/views/provider/admin/dashboards/_apiap_search_bar.html.slim

```diff
@@ -0,0 +1,20 @@
+div id=id
+
+javascript:
```

Comment:
It's better to move this to the pack, and send the api collection, dom class, etc with data props.

Reply:
I don't like the data-props approach in this case since we can end up having huge amount of data in the HTML, say there are 200 apis, the element would be ridiculously big, WDYT?

Reply:
Having the JS in the view is something we should avoid, is not testable and prone to bugs. It's not even transpiled. And no worries about the 200 elements, after my PR you could use the decorator and those will be shallow.